packages/terra-scrollable-container/docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Terra Scrollable Container
+
+The terra-scrollable-container component scrollable appropriate styling for standard scrolling css and a refCallback.
```

Comment:
Reword this description. Instead of saying that it applies styling, say that it's a content view that hides data accessible with scrolling.